apiBase/errors/ValidationError.js

```diff
@@ -1,20 +1,19 @@
 import FakeError from './FakeError';
 
-const msgText = (apiName, errors, model) =>
-  `${ apiName } had errors in ${ errors.join(',') } with properties ${ JSON.stringify(model.toJSON()) }`;
+const msgText = (api, errors) => `${api} had errors in ${errors.join(',')}`;
 
 export default class ValidationError extends FakeError {
-  constructor (apiName, model, errors) {
+  constructor (apiName, errors, status) {
```

Comment:
Yea, so i remove model because I can imagine this being used for things that don't have a model. The `msgText` thing can definitely be modified to be more helpful though. I do want to give devs access to the raw errors though.